pkgs/build-support/vm/default.nix

```diff
@@ -1687,6 +1687,10 @@ rec {
             url = mirror://ubuntu/dists/trusty/universe/binary-amd64/Packages.bz2;
             sha256 = "558637eeb8e340b871653e2060effe36e064677eca4eae62d9e4138dd402a610";
           })
+          (fetchurl {
+            url = mirror://ubuntu/dists/trusty-updates/main/binary-amd64/Packages.bz2;
```

Comment:
Nope. But neither are most of the other package urls in that file.

I take it as a sign that distros generally don't provide stable URLs for their package sets, but I didn't check.